### ISSUE TITLE
Missing queryStringParameters property in APIGatewayProxyWebsocketHandlerV2

### ISSUE THREAD

**Issue by pedroSoaresll** [DefinitelyTyped/DefinitelyTyped#68039]
Hey folks, I believe it is missing a property `queryStringParameters` when the WebSocket event is in a lambda function.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0730ea33dbf53280882d768648a6e8fa98993270/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L30

Check the image below which is a sample of a cloud watch log from a `$connect` event

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/11558773/344d046d-c52c-41a0-b186-814a77314efe)

We are missing those two properties, `queryStringParameters` and `multiValueQueryStringParameters `.

**Comment by brian-bourdon** [comment]
yeah i noticed that too, headers and multiValueHeaders are missing too
